insights/core/__init__.py

```diff
@@ -694,17 +737,23 @@ def test_all_parsers(logstamp):
                 if logstamp >= timestamp:
                     # Later - include
                     including_lines = True
-                    yield line
+                    if parsed_line:
+                        yield parsed_line
+                    else:
+                        yield self._parse_line(line)
```

Comment:
@xiangce I don't think you can make this change as it changes the default api for the 1.x branch.  It needs to return the same time as the input type to maintain compatibility.

Reply:
Sure, will revert them in the 1.x branch.